conf/application.conf

```diff
@@ -97,8 +97,7 @@ metrics {
 aws {
   s3 {
     region = "eu-west-2"
-    bucket.inbound = "ENTER YOUR INBOUND BUCKET NAME"
-    bucket.inbound = ${?AWS_S3_BUCKET_INBOUND}
+    bucket.inbound = "fus-inbound-776ea3bf11df5829827f7afb43c37174" 
```

Comment:
You've put bucket name into public repo :/ We have to change our naming scheme now as it has been revealed